lib/file.go

```diff
@@ -58,51 +58,77 @@ func pathKind(path string) string {
 	return "file"
 }
 
-// ReadDatasetFile decodes a dataset document into a Dataset
-func ReadDatasetFile(path string) (ds *dataset.Dataset, err error) {
-	var (
-		resp *http.Response
-		f    *os.File
-		data []byte
-	)
+// ReadDatasetFiles decodes a dataset document into a Dataset
+func ReadDatasetFiles(pathList []string) (*dataset.Dataset, error) {
```

Comment:
I'd prefer 
```go
func ReadDatasetFiles(pathList ... string) (*dataset.Dataset, error)
```
To me ReadDatasetFiles would be very useful when using qri-as-a-library by hiding the `[]string` from the caller:
```go
//  variadic arguments allows calls for easier single-file loading:
ds, err := lib.ReadDatasetFiles("./path.json")

//  and chaining that doesn't require []string allocation:
ds, err := lib.ReadDatasetFiles("./structure.json", "./meta.yaml")
```

Reply:
Good idea, this is much better.